renovate.json5

```diff
@@ -71,6 +71,7 @@
         "pinDigest",
         "rollback",
       ],
+      "excludeDepNames": ["stackit"],
```

Comment:
Maybe we can add a comment why we include it, or even a todo to revert this in the future?

Reply:
Thanks, done.